.travis.yml

```diff
@@ -9,6 +9,8 @@ install:
   - go get -v github.com/mattn/goveralls
 
 script:
+  # This unencryptes envvars.bash.enc into envvars.bash so that we can run the tests
+  - openssl aes-256-cbc -K $encrypted_263604b5b445_key -iv $encrypted_263604b5b445_iv -in envvars.bash.enc -out envvars.bash -d
```

Comment:
I have changed these vars to use a CFL_ prefix, as part of a refactoring to avoid confusion with cloudfront.  How do I reencrypt the envvars.bash to produce a new envvars.bash.enc?  Where do I get the required key(s)?  I couldn't find anything travis-related in too-many-secrets.